net-misc/curl/patches/curl-8.14.1.patchset

```diff
@@ -0,0 +1,36 @@
+From 48169f847fb18c1124f267c1cee861325971272c Mon Sep 17 00:00:00 2001
+From: Luc Schrijvers <begasus@gmail.com>
+Date: Wed, 16 Jul 2025 07:36:17 +0200
+Subject: Disable macros for gcc2
+
+
+diff --git a/include/curl/system.h b/include/curl/system.h
+index f1c2719..b49d9dd 100644
+--- a/include/curl/system.h
++++ b/include/curl/system.h
+@@ -336,8 +336,10 @@
+ #    define CURL_FORMAT_CURL_OFF_TU    "llu"
+ #    define CURL_SUFFIX_CURL_OFF_T     LL
+ #    define CURL_SUFFIX_CURL_OFF_TU    ULL
++#if GNUC > 2
```

Comment:
```suggestion
+#if __GNUC__ > 2
```
The same also below. (the comment from @waddlesplash was Markdown-formatted by Github).

Reply:
Running 32bit VM atm to check these things, will re-check and push/cleanup later when this is finished and tests are OK. Thanks!